src/api/subscriptionRemove.js

```diff
@@ -0,0 +1,12 @@
+import { apiDelete, Auth } from './apiFetch';
+
+export default (
+  auth: Auth,
+  subscriptions: string[],
```

Comment:
If you mean from 'subscriptionAdd' - that is correct. At least that is what the API expects (list of objects).